natalie.css

```diff
@@ -0,0 +1,252 @@
+
+
+header {
+  background-color: #A4243B;
+  text-align: center;
+  padding: 20px;
+  font-size: 40px;
+  color: lightsalmon;
+  
+}
+
+h2 {
+  text-align: center;
+  width: 100%;
+  font-size: 30px;
+}
+
+/* image to fade in overlay*/
+
+.intro {
+  position: relative;
+  width: 90%;
+}
+
+.intro_img {
+  display: block;
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  opacity: 0;
+  transition: .5s ease;
+  background-color: #008CBA;
+}
+
+.intro:hover .overlay {
+  opacity: 1;
+}
+
+.info {
+  color: white;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+/*overlay with tittle, good way to give explanation to the image*/
+.box {
+  position: relative;
+  width: 50%;
+  max-width: 300px;
+}
+
+.fav_image {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.over {
+  position: absolute; 
+  bottom: 0; 
+  background: rgb(0, 0, 0);
+  background: rgba(0, 0, 0, 0.5); /* Black see-through */
+  color: #f1f1f1; 
+  width: 100%;
+  transition: .5s ease;
+  opacity:0;
+  color: white;
+  font-size: 20px;
+  padding: 10px;
+  text-align: center;
+}
+
+.box:hover .over {
+  opacity: 1;
+}
+
+
+.intro_img {
+  text-align: center;
+  display: block;
+  height: 30%;
+  width: 30%;
+  object-fit: cover;
+ }
+
+  /* styling the body, the navbar, sections */
+
+body {
+   font-family: Arial, Helvetica, sans-serif;
+   background-color: #E7AD99;
+   margin: 30;
+  
```

Comment:
Look into a linter such as Prettier to automatically format your code for you. When formatting code, ensure no empty lines unless on purpose, syntax is consistent in placement and position too.